app/actions/UserActions.js

```diff
@@ -63,11 +63,17 @@ export function login(
     });
 }
 
+export function logoutWithRedirect() {
```

Comment:
This is certainly better than the previous one.. What do you think @ekmartin?

Reply:
👍